src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx

```diff
@@ -13,34 +13,64 @@ After you import OpenTelemetry data into New Relic, you can use a variety of too
 
-The opening page of the explorer is the **Summary** page listing various golden signals about your entity. Golden signals are key monitoring details such as response time, throughput, and error rate. By using this information, you can quickly decide if you need to dig deeper.
+After you click on a service in the **Explorer** tab, you see the **Summary** page listing various golden signals about your entity. Golden signals are key monitoring details such as response time, throughput, and error rate. By using this information, you can quickly decide if you need to dig deeper.
 
 ### Distributed tracing [#tracing]
 
-When you access distributed tracing through the explorer, you are looking at traces that include that service. Once you’re in that service, you can filter spans to find the ones you want. For example, to query `service.name` or `trace.id`, you can use the following:
+In **Distributed tracing**, you can locate traces and examine span details:
+
+1. To narrow the search for traces you want, you can run queries like these in the filter bar:
+    * `service.name = YOUR_SERVICE_NAME`
+    * `trace.id = YOUR_TRACE_ID`
+2. When you find an interesting trace, click on it to display a waterfall diagram showing the spans of the trace. 
+3. Click on specific spans within the trace to display span details in the right panel.
+
+<Callout variant="tip">
+For more ways to filter traces, see our [distributed tracing UI page](/docs/understand-dependencies/distributed-tracing/ui-data/understand-use-distributed-tracing-ui).
+</Callout>
+
+#### View spans with errors
+
+After you click on a span in the trace waterfall view, you can see span errors in the right panel under **Error details**.
+
+![Screenshot showing the right pane with Error details section](./images/view-span-errors.png "Screenshot showing the right pane with Error details section")
 
-* `service.name = YOUR_SERVICE_NAME`
-* `trace.id = YOUR_TRACE_ID`
+The error details are populated by spans containing `otel.status_code = ERROR` and display the content of `otel.status_description`.
 
-For more ways to filter and analyze your spans, see our [distributed tracing UI page](/docs/understand-dependencies/distributed-tracing/ui-data/understand-use-distributed-tracing-ui).
+To narrow your search for spans with errors, you can enter `otel.status_code = ERROR` directly in the distributed tracing filter bar.
 
+#### View span events
+
+If you send span events as described by the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#add-events), you can view them in the New Relic UI. Span events have two general types:
+  * Exceptions 
+  * Non-exceptions (for example, logs)
+
+If you have span events, links for these appear in the right pane:
+
+1. Click on a span in the waterfall view.
+2. In the right pane, click **View span events**, or you can expand **Error details** and click the link at the bottom of the errors.
```

Comment:
maybe consider changing "or you can" to just "or expand" to keep it sounding like a procedure

Reply:
@barbnewrelic, fixed it!